packages/vest/src/core/suite/getState/index.js

```diff
@@ -1,4 +1,4 @@
-import { getSuite } from '..';
+import { getSuite } from '../../state';
 
 /**
  * Retrieves most recent suite state.
```

Comment:
The function in this file should probably be renamed.

Reply:
but I don't know what name is correct get is busy in several files